src/nft-card-back.ts

```diff
@@ -381,44 +416,44 @@ export class NftCardBackTemplate extends LitElement {
     `
   }
 
-  private formatTrait(trait) {
-    return trait.replace(/_/g, ' ')
-  }
-
-  private buildTraits(traitData) {
+  private buildTraits(traitData: TraitData) {
     this.traits = {
       props: [],
       stats: [],
       rankings: [],
       boosts: []
     }
-    const {traits, collectionTraits} = this.traitData
+    const {traits: assetTraits, collectionTraits} = traitData
 
-    for (const trait of traits) {
+    for (const trait of assetTraits) {
       const type = this.getTraitType(trait, collectionTraits)
```

Comment:
what if `getTraitType`'s return type was `keyof Traits`? then you wouldn't have to add an `'s'` below and typescript would prevent errors when setting keys' values

Reply:
(see slack)